pkg/proxy/nftables/proxier.go

```diff
@@ -185,6 +185,12 @@ type Proxier struct {
 
 	// staleChains contains information about chains to be deleted later
 	staleChains map[string]time.Time
+
+	// numgen defines the algorithm for load-balancing using the ntfables "numgen" module.
+	// It can be "random", which will be about the same as the old "iptables" proxier,
+	// or "inc", which will give a round-robin distribution similar to the "ipvs"
+	// proxier with the "rr" scheduler
+	numgen string
```

Comment:
is the plan to make this a configuration option like in ipvs?

```
	// scheduler is the IPVS scheduler to use
	Scheduler string `json:"scheduler"`
```

Reply:
No plan at the moment. I just read the nft doc's and saw that it can be done. Easily.

But I can't say I have any strong opinions about it, except that is _may_ make some current users of ipvs more comfortable with the upgrade as mentioned in the description. But people may select ipvs for other reasons than "rr", e.g. get rid of the "probability chain" in iptables, that can look horrible for may endpoints, or simply that they know ipvs and it's easier to understand (my case). Or they want another scheduler, which can't be supported by nftables anyway.

Reply:
But yes, my thought was a config option.